tools/depends/target/python3/darwin_embedded.patch

```diff
@@ -268,7 +269,7 @@
      /* If we get here it's definitely an error */
  
      posix_path_error(path);
-@@ -5416,7 +5442,7 @@
+@@ -5413,7 +5439,7 @@
```

Comment:
Is the current range information incorrect?

Reply:
Yes